core/trino-main/src/test/java/io/trino/connector/MockConnector.java

```diff
@@ -261,6 +261,15 @@ public boolean schemaExists(ConnectorSession session, String schemaName)
             return listSchemaNames.apply(session);
         }
 
+        @Override
+        public void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties, TrinoPrincipal owner) {}
+
+        @Override
+        public void renameSchema(ConnectorSession session, String source, String target) {}
+
+        @Override
+        public void dropSchema(ConnectorSession session, String schemaName) {}
```

Comment:
Are there any other like these? Alter views? Let's implement as much as possible so don't need to go back and forth.

Reply:
In case of `VIEW` I guess we can address them in a separate PR